pythonforandroid/util.py

```diff
@@ -3,18 +3,17 @@
 from os import getcwd, chdir, makedirs, walk, uname
 import sh
 import shutil
-import sys
 from fnmatch import fnmatch
 from tempfile import mkdtemp
-try:
+
+# This Python version workaround left for compatibility during initial version check
```

Comment:
Nice to have comments about this. Best would be some unit tests. I can deal with this later